scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>"a set of workflow ids" -&gt; "a workflow ID and IDs of sub-workflows (if any)"</p><p> </p><p>"datas" -&gt; "estimated dates when the job was submitted and successfully finished"</p><p> </p><p>In a sense this is my mistake: I should have provided the docstring when constructing this class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>